rust/src/connection/network/transmitter/rpc.rs

```diff
@@ -100,9 +101,15 @@ impl RPCTransmitter {
             request = request_source.recv() => request,
             _ = shutdown_signal.recv() => None,
         } {
+            trace!("RPC dispatcher loop received request {:?}", request);
```

Comment:
If we want to see everything happening, on TRACE we'll basically see every message in and out